lib/task.js

```diff
@@ -309,17 +309,24 @@ function factory(
         var self = this;
 
         self.instantiateJob();
+        // Support schedulerOverrides for backwards graph definition compatibility
+        var timeout = self.definition.options.timeout;
```

Comment:
I have a concern that there may be conflict in further if we choose the name `timeout`, this is a common word, it is likely to be an options for a job (such as the ssh command job).
So maybe we can copy the python design, use `__` as prefix if the variable is reserved for RackHD, so `timeout` ==> `__timeout`
How do you think?